SpectraLogic.SpectraRioBrokerClient.Test/SpectraRioBrokerClientTest.cs

```diff
@@ -72,8 +72,8 @@ public void ArchiveTest()
             Assert.AreEqual(JobType.ARCHIVE, job.JobType);
             Assert.AreEqual(1, job.NumberOfFiles);
             Assert.AreEqual(1234, job.TotalSizeInBytes);
-            Assert.AreEqual("2018-01-23T03:52:46.869Z[UTC]", job.CreationDate);
-            Assert.AreEqual("2018-01-23T03:52:46.869Z[UTC]", job.LastUpdated);
+            Assert.AreEqual("1/23/2018 3:52:46 AM", job.CreationDate.ToString());
```

Comment:
I'm confused by this updated date format.  Why is it different?

Reply:
before we expected the string value that returns from Rio, now we expected the ToString that DateTime return

Reply:
Oh I see.  Is it still parsing the correct format from Rio though?

Reply:
The actual parsing occurred in the new extension function